src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -176,7 +179,11 @@ def dump_yaml_template(
             session, phylo_run.pathogen, phylo_run.template_args, group
         )
         builder: TemplateBuilder = TemplateBuilder(
-            phylo_run.tree_type, phylo_run.group, resolved_template_args, **context
+            phylo_run.tree_type,
+            phylo_run.pathogen,
```

Comment:
This is a straight bugfix. The TemplateBuilder init args expect: `tree_type, pathogen, group, template_args, **kwargs`. Somehow we've been missing passing the `pathogen` to the TemplateBuilder. I'd think that would have pretty major ramifications though, so I'm surprised this didn't get noticed sooner.

Reply:
Whoops, I got myself a bit confused. This is still a straight bugfix, but it's fixing a seldom used develpment function `dump_yaml_template` that we can use for only generating the build config yaml. We hadn't used it for awhile and it never got updated while the TemplateBuilder usage where it mattered (later in this same file) did get updated. I forgot that context when I wrote the above comment. But that's why there were no major ramifications and it didn't get noticed: it almost never gets used, and I just wound up using it for this work since a lot of my work was around fixing up the yaml config.

Reply:
`dump_yaml_template` was intended for use during development. nice catch!